src/main/kotlin/no/nav/pdfgen/Bootstrap.kt

```diff
@@ -146,7 +125,8 @@ fun render(applicationName: String, template: String, templates: Map<Pair<String
     val html = HANDLEBARS_RENDERING_SUMMARY.startTimer().use {
         templates[applicationName to template]?.apply(Context
                 .newBuilder(jsonNode)
-                .resolver(JsonNodeValueResolver.INSTANCE)
+                .resolver(JsonNodeValueResolver.INSTANCE,
+                        MapValueResolver.INSTANCE)
```

Comment:
FYI: Denne trengs for å kunne aksessere `@index` i templates, av en eller annen grunn. Funnet her: jknack/handlebars.java#577

Det bør sjekkes at alle templates fungerer med den Resolveren i bruk.